tests/Wallabag/CoreBundle/WallabagCoreTestCase.php

```diff
@@ -48,11 +48,12 @@ public function resetDatabase(KernelBrowser $client)
             'command' => 'doctrine:schema:drop',
             '--no-interaction' => true,
             '--force' => true,
+            '--full-database' => true,
             '--env' => 'test',
         ]), new NullOutput());
 
         $application->run(new ArrayInput([
-            'command' => 'doctrine:schema:create',
+            'command' => 'doctrine:migrations:migrate',
```

Comment:
The migrate command is creating the schema as well?

Reply:
yes, it's is goal. in fact in apps either you use `doctrine:schema:*` commands, either `doctrine:migrations:*` to manage database schema.
see the [very first migration](https://github.com/wallabag/wallabag/blob/master/app/DoctrineMigrations/Version20160401000000.php)